app/models/user.rb

```diff
@@ -3,6 +3,16 @@ class User < ApplicationRecord
   USERS_PARAMS = %i(name email password password_confirmation).freeze
 
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+    foreign_key: :follower_id,
+    dependent: :destroy,
+    inverse_of: :follower
```

Comment:
phải đặt foreign_key: :follower_id là vì Active Record mặc định đặt khoá ngoại dựa trên convention, VD user has many active_relationships thì khoá ngoại sẽ là user_id, tuy nhiên trong trường hợp sẽ đặt là follower_id (có thể hiểu là Id của người ấn Follow người khác)

inverse_of là để đánh dấu rằng các đối tượng liên quan có thể được truy cập từ một trong hai hướng. Ở đây, user has_many active_relationships và ngược lại relationships belongs to user. Ở trong TH này, khi khai báo inverse_of giữa 2 đối tượng có liên kết has many - through, sẽ có thể tự động tạo ra các đối tượng đã được liên kết, khi save 1 đối tượng